packages/bazel/src/collection/lib/index.ts

```diff
@@ -0,0 +1,15 @@
+import {branchAndMerge, chain, mergeWith, Rule,} from '@angular-devkit/schematics';
+
+import {validateLibSchema} from '../../../../schematics/src/collection/lib';
```

Comment:
even though it happened to work right now, this won't work in a prod env. bazel and schematics are too separate packages.

So you will have to import it using `@nrwl/schematics`.

I think it might be easer to just duplicate this functionality